content/agent/prometheus.md

```diff
@@ -15,11 +15,52 @@ further_reading:
 
 ## Overview
 
-This page first looks at the `PrometheusCheck` interface, and then proposes a simple Prometheus Check that collects timing metrics and status events from [Kube DNS](https://github.com/DataDog/integrations-core/blob/master/kube_dns/datadog_checks/kube_dns/kube_dns.py).
+This page first looks at the generic `Prometheus` check, the fastest & simpliest way to scrap custom metrics off prometheus endpoints and then dives in the `PrometheusCheck` interface for more advanced usage finishing by an example of writting a simple Check that collects timing metrics and status events from [Kube DNS](https://github.com/DataDog/integrations-core/blob/master/kube_dns/datadog_checks/kube_dns/kube_dns.py).
```

Comment:
* Prefer the word _and_ to  ampersand.
* "simplest" not simpliest; though I prefer "most simple".
* "scrape" not "scrap" (hello fellow French-speaker 😉).
* Be careful to capitalise proper names such as Prometheus.
* Dives _into_, not just _in_.
* Writing has one _t_.
* Is _Check_ capitalised?

Finally, this is a run on sentence that starts with the first comma and gets nasty towards "finishing". Suggest something like:
> This page covers the generic `Prometheus` check - which is the fastest and most simple way to scrape custom metrics from Prometheus endpoints - including a deep dive into the `PrometheusCheck` interface. There is also an example of how to write a simple check that collects timing metrics and status events from Kube DNS.